src/main/resources/templates/db_scripts/schema.json

```diff
@@ -98,7 +105,7 @@
     },
     {
       "tableName": "fiscal_year",
-      "fromModuleVersion": 1.0,
+      "fromModuleVersion": "mod-finance-storage-4.0.0",
```

Comment:
moving from 1.0 to 4.0?

Reply:
was anything in these tables updated?

Reply:
our pom is still at 3.1.0-SNAPSHOT? will these changes be applied in-spite of that?
Also, I thought we wanted to increase the module version with any changes immediately and not during the release? aren't we following that?

Reply:
I think this is all backwards compatible, so we should be ok w/o major version changes

Reply:
I meant a changes compared to [version 3.0.0](https://github.com/folio-org/mod-finance-storage/compare/v3.0.0...736421a0cd8353e289a69928a4ede7b3d13d0694).